samples/scala-value-entity-customer-registry/src/main/scala/customer/Main.scala

```diff
@@ -0,0 +1,32 @@
+package customer
+
+import com.akkaserverless.scalasdk.AkkaServerless
+import customer.action.customer_action.CustomerActionImpl
+import customer.view.customer_view.CustomerByEmailView
+import customer.view.customer_view.CustomerByNameView
+import customer.view.customer_view.CustomerSummaryByNameView
+import customer.view.customer_view.CustomersResponseByNameView
+import org.slf4j.LoggerFactory
+
+object Main {
```

Comment:
this seems to be checking in some files that would be generated, AFAICS unchanged - wouldn't it be more convenient to leave them out of git and have them generated when compiling?